argilla/tests/integration/test_export_dataset.py

```diff
@@ -202,3 +202,53 @@ def test_import_dataset_from_hub(
 
         assert new_dataset.settings.fields[0].name == "text"
         assert new_dataset.settings.questions[0].name == "label"
+
+    @pytest.mark.parametrize("with_records_import", [True, False])
+    def test_import_dataset_from_hub_using_settings(
+        self,
+        token: str,
+        dataset: rg.Dataset,
+        client,
+        mock_data: List[dict[str, Any]],
+        with_records_export: bool,
+        with_records_import: bool,
+    ):
+        repo_id = f"argilla-internal-testing/test_import_dataset_from_hub_with_records_{with_records_export}"
```

Comment:
I think the name should be u
```suggestion
        repo_id = f"argilla-internal-testing/test_import_dataset_from_hub_using_settings_with_records{with_records_export}"
```